mlrun/serving/remote.py

```diff
@@ -527,3 +527,59 @@ def post_init(self, mode="sync", **kwargs):
         self.fill_placeholders = self.fill_placeholders or False
         self.rundb = mlrun.get_run_db()
         self.url = self.rundb.get_base_api_url(self.path)
+
+
+class NuclioRemoteStep(RemoteStep):
+    """
+    Graph step implementation for invoking Nuclio functions remotely.
+    :param fn: Either an mlrun.runtimes.RemoteRuntime object or
+    a string URI in the form 'project/function'.
+    """
+
+    def __init__(
+        self,
+        fn: Union[mlrun.runtimes.RemoteRuntime, str, None] = None,
+        method: str = "POST",
+        **kwargs,
+    ):
+        super().__init__(url="", method=method, **kwargs)
+        self.rundb = None
+        self.fn = fn
+
+    def post_init(self, mode="sync", **kwargs):
+        self.rundb = mlrun.get_run_db()
+        if not self.fn:
+            raise mlrun.errors.MLRunRuntimeError(
+                f"Cannot find function '{self.fn}' in the MLRun DB. "
+                "Please verify that the function URI is correct and that it is stored properly."
+            )
+        if not isinstance(self.fn, (mlrun.runtimes.RemoteRuntime, str)):
+            raise mlrun.errors.MLRunInvalidArgumentTypeError(
+                "Parameter 'fn' must be of type mlrun.runtimes.RemoteRuntime or str."
+            )
+
+        if isinstance(self.fn, str):
```

Comment:
If that's possible, we need to ask the user to provide only the function name and inherit the project name from the client or the serving function spec